### ISSUE TITLE
Use module thumbnails in Jig Player and Jig Editor sidebars

### ISSUE THREAD

**Issue by dakom** [ji-devs/ji-cloud#1242]
They will exist at the following URL:

```
{remote_target.uploads_url()}/screenshot/{jig_id}/{module_id}/thumb.jpg
```

So for example:
https://uploads.sandbox.jicloud.org/screenshot/d46ca2d2-eeef-11eb-8c76-77f818ce2b73/1db6aa60-eef4-11eb-bfbe-4799d28bb7f0/thumb.jpg

(currently it says "No Auth" due to https://github.com/ji-devs/ji-cloud/issues/1241)

**Comment by MendyBerger** [comment]
Tried this with another jig/module and I got a 404, do I need to do something to trigger the thumbnail generation?

**Comment by MendyBerger** [comment]
Resolved in dc3e68754442947e47945252e6b4a46dc80c52fb
